crates/sui-benchmark/tests/simtest.rs

```diff
@@ -58,6 +58,7 @@ mod test {
     }
 
     #[sim_test(config = "test_config()")]
+    #[ignore = "The benchmark client aborts certificates submission when it fails to gather a quorum of acknowledgements. This happens upon epoch change."]
```

Comment:
Is this intended change?

Reply:
yes this is expected. I would need to re-write this test entirely but I though I would do it in a separate PR